README.md

```diff
@@ -12,19 +12,20 @@ Refer to the top section for docker/singularity image urls.
 ### Docker container
 
 ```bash
+# If using Spack-based docker
+# . /etc/profile.d/phasm_spack_environment.sh
+
 # start from PHASM root directory
 docker pull <docker_img>  # optional
 docker run -it -v ${PWD}:/app <docker_img>  # mount current directory
 
 mkdir build && cd build   # build
 cmake -DCMAKE_PREFIX_PATH="$DEPS/libtorch;$DEPS/JANA2/install" \
- -DLIBDWARF_DIR="$DEPS/libdwarf/installdir" -DPIN_ROOT="$DEPS/pin" ..
+    -DLIBDWARF_DIR="$DEPS/libdwarf/installdir" -DPIN_ROOT="$DEPS/pin" \  # not necessay for Spack
```

Comment:
What does using Spack mean for the CMake prefix path? I assume Spack wants all of its installed packages to live in `/opt/spack/COMPLICATED_HASH/...` and then put symlinks into `/usr/local` or whatever. I'd like to get rid of the weird things like `$DEPS/libdwarf/installdir` and be able to just set `CMAKE_PREFIX_PATH=path/to/spack/root` or similar

Reply:
This will be deleted. Spack offers a "viewpoint"  at `/opt/view` and a script to activate the envs so that CMAKE_PREFIX_PATH includes `/opt/view`.

I am still in the progress of meeting all the items list on top. Last two weeks are heavy-coding and these two weeks I prefer more literature reading.